src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/BindingInfo.cs

```diff
@@ -125,13 +125,7 @@ public CompositePropertyFilterProvider(IEnumerable<IPropertyFilterProvider> prov
                 _providers = providers;
             }
 
-            public Func<ModelMetadata, bool> PropertyFilter
-            {
-                get
-                {
-                    return CreatePropertyFilter();
-                }
-            }
+            public Func<ModelMetadata, bool> PropertyFilter => CreatePropertyFilter();
```

Comment:
This isn't right (old code isn't right). This should cache (not that anyone uses this feature)